code/game/objects/items/devices/radio/radio.dm

```diff
@@ -231,8 +231,14 @@ var/global/list/default_medbay_channels = list(
 /obj/item/device/radio/proc/autosay(var/message, var/from, var/channel) //BS12 EDIT
 	var/datum/radio_frequency/connection = null
 	if(channel && channels && channels.len > 0)
-		if (channel == "department")
-			channel = channels[1]
+		if(channel == "department")
+			for(var/freq in channels)
+				if(freq == "Common" || freq == "Entertainment")
+					continue
+				channel = freq
+				break
+			if(channel == "department") // didn't find one, use first one
+				channel = channels[1]
```

Comment:
What is this for?

Reply:
back-up, if it doesn't have non-common or non-entertainment channels, it'll use the first channel in its list, which is usually common